### DISCUSSION TITLE
"xxx" Wants to Use "google.com" to Sign In

### DISCUSSION THREAD

**Discussion by RocMark** [expo/expo#19751] [Archived: Questions]
is there a way to modify "xxx" now i am getting "app" not sure if this is default value

here is the only issue i found https://stackoverflow.com/questions/59493957/null-wants-to-use-google-com-to-sign-in
but i dont know how to implement it with expo
<img width="406" alt="image" src="https://user-images.githubusercontent.com/33424161/198492107-9d3c2f80-ec7e-49d0-b397-010acfed7290.png">

**Comment by raajnadar** [comment]
You need to follow this for ios, even I had this issue in the past

https://stackoverflow.com/a/50604942/5519872
